app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :user_id, :book_id, :from, :to
```

Comment:
You can use belongs_to :user and belongs_to :book to add that info if you have to return it instead of the ids